app/tip/page.tsx

```
@@ -1,17 +1,489 @@
+      attester
+    }
+  }
+`;
+const formSchema = z.object({
+  comment: z.string(),
+  amount: z
+    .string()
+    .regex(/^\d*\.?\d*$/, "Please enter a valid number")
+    .refine((val) => {
+      const num = Number(val);
+      return !isNaN(num) && num > 0;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)